software/coremark/core_portme.c

```diff
@@ -1,8 +1,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include "coremark.h"
-#include "platform.h"
-#include "encoding.h"
+//#include "platform.h"
+//#include "encoding.h"
```

Comment:
If we still want coremark to build with the legacy BSP, these should probably still be included somehow.

Reply:
I have no idea what is going on with freedom-e-sdk.  I just see breaking changes being added without any apparent plan, and on master instead of on a branch.  So I have no idea how to fix this.  But my patch does at least let me build some broken stuff using BSP=mee which is apparently the intent of the breaking changes.

Reply:
"If we still want coremark to build with the legacy BSP"

I don't know. Do we? Long term, shouldn't MEE be replacing the legacy BSP?

Now, when MEE doesn't work, certainly. Which makes it disappointing that WIP stuff for MEE is getting checked in on master branch and breaking it. Normal people actually use freedom-e-sdk. Why isn't this work happening on a branch? Telling people "Check out revision baeeb8f from September" (or something) isn't the right way to do it,. Master should simply work. Always.

@palmer-dabbelt

Reply:
Is legacy broken? All the MEE changes should still be isolated behind `BSP=mee`. Either way you're right, a development branch is a good idea.

I made the original comment here because this change would be mutually exclusive between legacy and MEE.